content/en/runtime.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Runtime
+status: Feedback Appreciated
+category: concept
+---
```

Comment:
```suggestion
tags: ["", "", ""]
---
```
We'll need to add the appropriate tags from [this list](https://glossary.cncf.io/style-guide/#tags)

Reply:
I suggest `application`